src/NodaTime/ZonedDateTime.cs

```diff
@@ -441,6 +445,7 @@ public override int GetHashCode()
         /// <returns>A value less than zero if the instant represented by this zoned date/time is earlier than the one in
         /// <paramref name="other"/>; zero if the instant is the same as the one in <paramref name="other"/>;
         /// a value greater than zero if the instant is later than the one in <paramref name="other"/>.</returns>
+        [Obsolete("Use ZonedDateTime.Comparer.Instant to compare values by instant for compatibility with 2.0")]
```

Comment:
No, because they were already only comparing by instant. The extra part of the comment for the operators was because comparing by instant wouldn't provide the exact same behaviour.